tests/consensus_tests/test_named_vector_crud.py

```
@@ -158,6 +158,140 @@ def test_create_vector_no_optimization(tmp_path: pathlib.Path):
         assert "new_sparse" in sparse, f"new_sparse not in sparse vectors config on {uri}"
 
 
+def update_collection_hnsw_m(peer_url, collection_name, m, timeout=30):
+    """Force a config-mismatch by changing the collection-wide HNSW `m`.
+
+    Returns immediately (wait happens via collection status); the config-mismatch
+    optimizer then rebuilds every already-indexed segment to apply the new value.
+    """
+    r = requests.patch(
```

Ruff (0.15.20)

[error] Probable use of `requests` call without timeout

(S113)